backend/internal/db/league.go

```diff
@@ -12,7 +12,7 @@ func (s *session) GetLeagues(sortField, sortOrder string) ([]models.League, erro
 	}
 
 	leagues := make([]models.League, 0)
-	err := s.Order(sortField + " " + sortOrder).Find(&leagues)
+	err := s.Order(sortField + " " + sortOrder).Preload("Teams").Find(&leagues)
```

Comment:
Leagues.Teams.League shows in the response, but is all null values. This is just informational, but I wonder if we can remove the inner League from showing.

```
{
  "status_code": 200,
  "status_string": "OK",
  "request_id": "044bd6fe-656f-4659-8f06-a46e700ea9fb",
  "message": "",
  "response_data": [
    {
      "id": 1,
      "created_at": "2024-06-14T02:36:18.708017Z",
      "updated_at": "2024-06-14T02:36:18.708017Z",
      "correlation_id": "bc5dae02-7519-46f0-b327-6a6e5a725e4d",
      "season_id": 1,
      "name": "Aperiam",
      "teams": [
        {
          "id": 1,
          "created_at": "2024-06-14T02:36:18.727929Z",
          "updated_at": "2024-06-14T02:36:18.727929Z",
          "correlation_id": "aa1959f7-e302-4c32-9615-54c4098c4a6f",
          "name": "Aliquid",
          "logo_id": "",
          "color": "#8F6B71",
          "league_id": 1,
          "league": {
            "id": 0,
            "created_at": "0001-01-01T00:00:00Z",
            "updated_at": "0001-01-01T00:00:00Z",
            "correlation_id": "",
            "season_id": 0,
            "name": "",
            "teams": null
          },
          "roster": null,
          "roster_id": null,
          "wins": 0,
          "losses": 0
        },
        {
          "id": 2,
          "created_at": "2024-06-14T02:36:18.735073Z",
          "updated_at": "2024-06-14T02:36:18.735073Z",
          "correlation_id": "543e5bc0-c9f8-4908-a22b-a478e6009817",
          "name": "Tempora",
          "logo_id": "",
          "color": "#064A7B",
          "league_id": 1,
          "league": {
            "id": 0,
            "created_at": "0001-01-01T00:00:00Z",
            "updated_at": "0001-01-01T00:00:00Z",
            "correlation_id": "",
            "season_id": 0,
            "name": "",
            "teams": null
          },
          "roster": null,
          "roster_id": null,
          "wins": 0,
          "losses": 0
        },
        {
          "id": 3,
          "created_at": "2024-06-14T02:36:18.740079Z",
          "updated_at": "2024-06-14T02:36:18.740079Z",
          "correlation_id": "3ee278ec-c5a6-40ff-84e9-d548b7ee2ae5",
          "name": "Aut",
          "logo_id": "",
          "color": "#F20B97",
          "league_id": 1,
          "league": {
            "id": 0,
            "created_at": "0001-01-01T00:00:00Z",
            "updated_at": "0001-01-01T00:00:00Z",
            "correlation_id": "",
            "season_id": 0,
            "name": "",
            "teams": null
          },
          "roster": null,
          "roster_id": null,
          "wins": 0,
          "losses": 0
        },
        {
          "id": 4,
          "created_at": "2024-06-14T02:36:18.74446Z",
          "updated_at": "2024-06-14T02:36:18.74446Z",
          "correlation_id": "83c310a5-1ad7-47a4-9652-871df3a83eaa",
          "name": "In",
          "logo_id": "",
          "color": "#11AD76",
          "league_id": 1,
          "league": {
            "id": 0,
            "created_at": "0001-01-01T00:00:00Z",
            "updated_at": "0001-01-01T00:00:00Z",
            "correlation_id": "",
            "season_id": 0,
            "name": "",
            "teams": null
          },
          "roster": null,
          "roster_id": null,
          "wins": 0,
          "losses": 0
        }
      ]
    },

...
```

Reply:
I agree it would be nice if we could remove some values from showing. It is also happening with the Season data when the teams are loaded.